VimeoNetworking/Sources/Models/Subscription.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  self.swift
+//  Pods
+//
+//  Created by Lim, Jennifer on 2/8/17.
+//
+//
+
+import Foundation
+
+public class Subscription: VIMModelObject
+{
+    // MARK: - Properties
+    
+    public var comment: NSNumber?
+    public var credit: NSNumber?
+    public var like: NSNumber?
+    public var mention: NSNumber?
+    public var reply: NSNumber?
+    public var follow: NSNumber?
+    public var videoAvailable: NSNumber?
+    public var vodPreorderAvailable: NSNumber?
+    public var vodRentalExpirationWarning: NSNumber?
+    public var accountExpirationWarning: NSNumber?
+    public var share: NSNumber?
+    
+    public var toDictionary: [NSObject: AnyObject]
```

Comment:
Why a `var` here? Did they compiler complete with a `let`? It also looks like all of our keys are just strings, so could we change the type to `[String: AnyObject]`?